pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -95,6 +92,16 @@ func (i *CommandInitOption) Validate(parentCommand string) error {
 
 // Complete Initialize k8s client
 func (i *CommandInitOption) Complete() error {
+	// check config path of host kubernetes cluster
+	env := os.Getenv("KUBECONFIG")
+	if env != "" {
+		i.KubeConfig = env
+	}
```

Comment:
Does this mean `KUBECONFIG` environment would override `--kubeconfig`? 

The kubectl get config according to following rules, that we can't violate :
  1. Via the command-line flag --kubeconfig
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

Reply:
Ok, I will fix it as these rules.